conf.py

```diff
@@ -0,0 +1,53 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
```

Comment:
It might be worth adding the same css overide file, favicon e.t.c. in here so that this matches the scipp documentation as closely in style as possible. 

I think the additions we probably want some variation of the following bits:
 * https://github.com/scipp/scipp/blob/master/docs/conf.py#L29
 * https://github.com/scipp/scipp/blob/master/docs/conf.py#L127
* https://github.com/scipp/scipp/blob/master/docs/conf.py#L143
* https://github.com/scipp/scipp/blob/master/docs/_templates/layout.html
* https://github.com/scipp/scipp/blob/master/docs/_static/theme_overrides.css

Reply:
The only thing not done was to use the scipp logo.  Or at least I did it then reverted the change. I'm not sure that fully makes sense for these data reduction notebooks to have the logos.